erpc.dist.yaml

```
@@ -93,6 +93,20 @@ projects:
             backoffMaxDelay: 10s
             backoffFactor: 0.3
             jitter: 500ms
+      - id: tenderly-chain-example-1
+        endpoint: https://mainnet.tenderly.co/xxxxxxxxxxxxxxxxxxxxxx
```

```suggestion
      - id: tenderly-example-1
        endpoint: tenderly://YOUR_TENDERLY_API_KEY
```

We could suggest the provider in this example